include/openthread/nat64.h

```diff
@@ -87,6 +87,67 @@ typedef struct otIp4Cidr
     uint8_t      mLength;
 } otIp4Cidr;
 
+/**
+ * Sets the CIDR block used for the source address of the translated address. A valid CIDR must have a non-zero prefix
+ * length. Note: The actual addresses used in the CIDR is limited by the size of mapping pool.
+ *
+ * The NAT64 translator will expire all existing sessions when the provided CIDR is valid and is not the one configured.
+ *
+ * This function is available only when OPENTHREAD_CONFIG_NAT64_MANAGER_ENABLE is enabled.
+ *
+ * @param[in] aInstance A pointer to an OpenThread instance.
+ * @param[in] aCidr A pointer to an otIp4Cidr for the IPv4 CIDR block for NAT64.
+ *
+ * @retval  OT_ERROR_INVALID_ARGS   The given CIDR is not a valid IPv4 CIDR for NAT64.
+ * @retval  OT_ERROR_NONE           Successfully set the CIDR for NAT64.
+ *
+ * @sa otBorderRouterSend
+ * @sa otBorderRouterSetReceiveCallback
+ * @sa otBorderRouterSetNat64TranslatorEnabled
+ *
+ */
+otError otBorderRouterSetIp4CidrForNat64(otInstance *aInstance, const otIp4Cidr *aCidr);
```

Comment:
It's better to rename `otBorderRouterSetIp4CidrForNat64`, `otBorderRouterSetNat64TranslatorEnabled` and `otIp6NewMessageForNat64` to be like `otNat64xxxx` so that they follow the APIs' naming convention. 

Another option is moving `otBorderRouterSetIp4CidrForNat64`, `otBorderRouterSetNat64TranslatorEnabled` to `border_router.h`.